lib/jsonapi/serializer/dsl.rb

```diff
@@ -81,9 +81,9 @@ def attributes(*attributes_list, **options, &block)
       alias attribute attributes
 
       def belongs_to(relationship_name, options = {}, &block)
```

Comment:
this is kindof problematic previously because this returns all relationships defined in serializer instead of just the included object